src/doc/book/testing.md

```diff
@@ -499,6 +499,10 @@ be imported in every test with `mod common;`
 That's all there is to the `tests` directory. The `tests` module isn't needed
 here, since the whole thing is focused on tests.
 
+Note, when building integration tests, cargo will not pass the test flag to the
```

Comment:
That's referring to `--test`, which is a command line argument, also called a "flag"

https://en.wikipedia.org/wiki/Command-line_interface#Command-line_option